### PR TITLE
doc/troubleshooting: edit cpu-profiling.rst

### PR DESCRIPTION
**Pull request by zdover23**
Improve the language in the "oprofile" section of the RADOS Troubleshooting Guide.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/troubleshooting/cpu-profiling.rst [hunk 1]
```diff
@@ -9,59 +9,72 @@ you can profile Ceph's CPU usage. See `Installing Oprofile`_ for details.
 Initializing oprofile
 =====================
 
-The first time you use ``oprofile`` you need to initialize it. Locate the
-``vmlinux`` image corresponding to the kernel you are now running. :: 
+``oprofile`` must be initalized the first time it is used. Locate the
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/cpu-profiling.rst:12]: I never know whether to capitalize such cases

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/troubleshooting/cpu-profiling.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/troubleshooting/cpu-profiling.rst:12]: I have a strong intuition that the command or program name should be kept as it is, as here. Cole fights like hell to find a way to put a word or phrase before commands or program names like this.

Comment (anthonyeleven) [reply doc/rados/troubleshooting/cpu-profiling.rst:12]: That's my usual practice as well.

Comment (anthonyeleven) [reply doc/rados/troubleshooting/cpu-profiling.rst:12]: >The ``oprofile`` system must be initialized ...

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/53059 - Reef backport
https://github.com/ceph/ceph/pull/53060 - Quincy backport

**Pull request merged**